modules/vba_documents/app/controllers/vba_documents/v2/uploads_controller.rb

```diff
@@ -82,11 +84,12 @@ def submit
         upload_model = UploadFile.new
         begin
           upload_model.multipart.attach(io: StringIO.new(request.raw_post), filename: upload_model.guid)
+          upload_model.metadata['version'] = 2
           upload_model.save!
           parts = VBADocuments::MultipartParser.parse(StringIO.new(request.raw_post))
           inspector = VBADocuments::PDFInspector.new(pdf: parts)
           validate_parts(parts)
-          validate_metadata(parts[META_PART_NAME])
+          validate_metadata(parts[META_PART_NAME], submission_version: upload_model.metadata['version'])
```

Comment:
For upload_model.metadata['version'] it looks like the possible values are nil,1, or 2?

If we pass in: upload_model.metadata['version'].to_i, nil  => 0, 1 => 1, 2 =>2